### PR TITLE
Add management command for generating useful stats about Vumi Go.

### PR DESCRIPTION
**Pull request by hodgestar**
Sometimes people need to know how much stuff a Vumi Go instance is doing.

### CODE REVIEW

File: go/base/management/commands/go_system_stats.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                "A list of when conversations were started by type and"
+                " date.")),
+        make_command_option(
+            'message_counts_by_date',
+            help=(
+                "Inbound and outbound message counts and total unique users"
+                " by date and conversation type.")),
+    )
+
+    def handle_command_conversation_types(self, *args, **options):
```
Comment (smn) [inline go/base/management/commands/go_system_stats.py:56]: I think we're missing a `handle_*` dispatch-y thing?

Comment (hodgestar) [inline go/base/management/commands/go_system_stats.py:56]: The  base class provides that. :D

Comment (smn) [inline go/base/management/commands/go_system_stats.py:56]: doh, /me makes note not to review during conf calls.

**Comment by hodgestar** [comment]
Ready for review.

**Comment by smn** [comment]
:+1: once Travis but brain kind of frazzled at this stage.

**Comment by hodgestar** [comment]
See https://boto.readthedocs.org/en/latest/ref/s3.html if anyone needs a boto S3 API reference.

**Pull request merged**